src/openrct2/actions/BannerPlaceAction.h

```diff
@@ -11,18 +11,12 @@
 
 #include "GameAction.h"
 
-class BannerPlaceActionResult final : public GameActions::Result
+struct BannerPlaceActionResult
 {
-public:
-    BannerPlaceActionResult();
-    BannerPlaceActionResult(GameActions::Status err);
-    BannerPlaceActionResult(GameActions::Status err, rct_string_id msg);
-    BannerPlaceActionResult(GameActions::Status err, rct_string_id title, rct_string_id message);
-
     BannerIndex bannerId = BANNER_INDEX_NULL;
 };
 
-DEFINE_GAME_ACTION(BannerPlaceAction, GameCommand::PlaceBanner, BannerPlaceActionResult)
+DEFINE_GAME_ACTION(BannerPlaceAction, GameCommand::PlaceBanner, GameActions::Result)
```

Comment:
Is `GameActions::Result` still necessary? Perhaps a new macro to remove this (or make it the default?) would be nicer.

Reply:
This is something I want to do once they are all refactored.